src/routes/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express'
+import productsRouter from './ProductsRoute'
+
+const router: Router = Router()
+router.use(productsRouter)
```

Comment:
There should be an endpoint inside the parenthesis something like this router.use('products',productsRouter) so when our app requests the products it knows where to route.